moviebot/dialogue_manager/dialogue_state.py

```diff
@@ -61,7 +62,7 @@ def _agent_offer_state(self):
         }
         return str([x for x, y in offer_state.items() if y])
 
-    def _dict(self) -> str:
+    def to_dict(self) -> Dict[str, Any]:
         """Prints the state to debug the position of the agent.
```

Comment:
Nit: this method does not print. Maybe the Docstring can be updated to something like: `Returns the dialogue state as a dictionary.`

Reply:
Done